### PR TITLE
*: add the error check

### PR DESCRIPTION
**Pull request by zimulala**
### What problem does this PR solve? 
Add the error check

### What is changed and how it works?
Return the error
Check the error information

### Check List 

Tests 

 - Unit test

Code changes

Side effects

Related changes

---
This change is [Reviewable](https://reviewable.io/reviews/pingcap/tidb/8770)

### CODE REVIEW

File: executor/executor.go [hunk 1]
```diff
@@ -1083,6 +1083,9 @@ func (e *MaxOneRowExec) Next(ctx context.Context, chk *chunk.Chunk) error {
 
 	childChunk := e.children[0].newFirstChunk()
 	err = e.children[0].Next(ctx, childChunk)
+	if err != nil {
+		return errors.Trace(err)
+	}
```
Comment (gregwebs) [inline executor/executor.go]: So that was an ignored error. A good demonstration of using the error check.

**Comment by gregwebs** [comment]
3 ignored errors! Thanks, LGTM, would love to have this running in the CI.

**Comment by shenli** [comment]
LGTM

**Comment by shenli** [comment]
@zimulala We need to cherry-pick this to the release branches.

**Pull request merged**